codechecks.yml

```diff
@@ -0,0 +1,4 @@
+checks:
+  - name: typecov
+    options:
+      - atLeast: 99
```

Comment:
[suggestion] Given TS is still relatively new for Open edX / Paragon, I'd suggest not enforcing any specific coverage targets quite yet. That is, at least to start, I'd like this be more of an inform on PRs rather than something that will block PRs from merging